crates/swc_ecma_transforms_proposal/tests/decorators.rs

```diff
@@ -10,7 +10,7 @@ use swc_common::{chain, comments::SingleThreadedComments, Mark};
 use swc_ecma_parser::{EsSyntax, Syntax, TsSyntax};
 use swc_ecma_transforms_base::{assumptions::Assumptions, resolver};
 use swc_ecma_transforms_proposal::{decorator_2022_03::decorator_2022_03, DecoratorVersion};
-use swc_ecma_transforms_testing::{test_fixture, FixtureTestConfig};
+use swc_ecma_transforms_testing::{test_module_fixture, FixtureTestConfig};
```

Comment:
Can we have `FixtureTestConfig.module: Option<bool>`, which defaults to `None`, and 

 - `Some(true)`: Module (and applies visitor after wrapping as Program)
 - `Some(false)`: Script (and applies visitor after wrapping as Program)
 - `None`: Program